sdk/nodejs/policy/policy.ts

```diff
@@ -24,6 +26,13 @@ export interface PolicyPackArgs {
      * The policies associated with a PolicyPack.
      */
     policies: Policies;
+
+    /**
+     * Indicates what to do on policy violation, e.g., block deployment but allow override with
+     * proper permissions. Default for all policies in the PolicyPack. Individual policies can
+     * override.
+     */
+    enforcementLevel?: EnforcementLevel;
```

Comment:
Hmm I am wondering if from a service perspective we should have the ability to set "undefined" (which wouldnt really be undefined),  but basically a way to say default to the all enforcement level instead of using the policy specific one.